### ISSUE TITLE
404 Not Found after installation 

### ISSUE THREAD

**Issue by Nico-ITA** [mettle/sendportal#238]
Im having this issue after normal installation on Ubuntu 20.4 . Also other issues closed didnt help

**Comment by Pushkraj19** [comment]
did you run installation via cli?

`php artisan sp:install
`

**Comment by andykaufseo** [comment]
same issue, 404 on nginx, root path changed to /public, installation executed via cli

**Comment by uhlhosting** [comment]
Same here, executed from CLI with artisan, there is nothing when I hit main domain, I must specify the url to /public/index.php to work. And are plenty errors in console:

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/6324047/213764041-59b8760c-4e55-4df7-8079-95e67ad3f02f.png">

**Comment by davidmartinrius** [comment]
same problem here. I configured it with an Nginx and PHP 8.1. In the same server I already host other laravels. The others work properly except this project. I already checked the configuration and it is right (ngonx, .env, permissions for www-data). Also I tried downgrading the php version to 7.3, but did not work neither...

Please, if you find out what happened and you know how to solve it, could you share it please?

**Comment by JonoB** [comment]
This looks like a hosting environment issue, which is not something that we can support

**Comment by davidmartinrius** [comment]
Great, you closed an issue taking for granted something you actually don't know and three people reported 👏

**Comment by uhlhosting** [comment]
@davidmartinrius we tried... I had step by step install, and all seem to be well, up to the point where the mechanism it was made is very dumb, since there is some sending pool or what ever, I could had not manage make it send anything. Big nice waste of time to try making it work, when we get here for help... Not much attention we got, must give a try to something else. I hosted it the app in a hosting platform with native support for laravel and it failed, what to think of other users who do not have this option.

**Comment by Number0ne** [comment]
So I'm going to write this to help others even though I still need help on it but it seems the http 404 error is an environment issue.

I'm using Apache on an ubuntu server VM and I set up the app with all the relevant php plugins/ extensions and installed it. Then I ran it on a test server using php -S and specifying the url. It flagged a lot of errors in the console which could be solved by changing the permissions of the GUO with chmod -R 766 would do for the entire folder or 777 if you just want to get everything setup and come back to this later (not recommended).

In the test server everything worked but then when I pointed Apache to the folder I started getting the 404 error but the strange thing is if I add index.php to the url then it works. 

https://myurl.com/index.php/login works but https://myurl.com/login doesn't work even though my settings should resolve the domain name to the index.php or index.html file.

@uhlhosting if you have any thoughts then it would be appreciated.
